src/components/withViewportOffsetTop.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import getComponentDisplayName from '../libs/getComponentDisplayName';
+import addViewportResizeListener from '../libs/VisualViewport';
+
+const viewportOffsetTopPropTypes = {
+    viewportOffsetTop: PropTypes.number.isRequired,
```

Comment:
Let's add a comment here explaining what this prop is, that way if anyone else wants to use this component it's very clear to them